assets/terraform-modules/packet/flatcar-linux/kubernetes/workers/cl/worker.yaml.tmpl

```diff
@@ -89,7 +66,27 @@ systemd:
             "%{~ if bgp_node_labels != "" ~}
             BGP_PEER_ADDRESS=$(ip route | grep '10.0.0.0/8' | awk {'print $3'}); \
             %{~ endif ~}
-            /usr/lib/coreos/kubelet-wrapper \
+            docker run \
+            --network=host \
```

Comment:
Does using `--network=host` copies `/etc/resolv.conf` from the host to the container?

Reply:
That is a default behaviour despite of the value of `--network` flag. From the docs:

> By default, a container inherits the DNS settings of the host, as defined in the `/etc/resolv.conf` configuration file.

source: https://docs.docker.com/config/containers/container-networking/#dns-services

Reply:
ah, TIL, thanks!